### ISSUE TITLE
VLC doesn't open

### ISSUE THREAD

**Issue by DerangedMemory** [rom1v/sndcpy#56]
Whenever I try using the program VLC doesn't open as I suspect it should, even though I have it installed.
This is what I see on my CMD prompt.
I am on Windows 10.
> Waiting for device...
> Performing Streamed Install
> Success
> Starting: Intent { act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] cmp=com.rom1v.sndcpy/.MainActivity }
> Press Enter once audio capture is authorized on the device to start playing...
> Playing audio...

**Comment by rom1v** [comment]
This is expected, it should play the audio in backgrojnd without openong the interface (remove `-Idummy` from the script if you want the interface).

If you have no sound, try VLC for Android or Youtube on your device to check.

**Comment by jeongpup** [comment]
> This is expected, it should play the audio in backgrojnd without openong the interface (remove `-Idummy` from the script if you want the interface).
> 
> If you have no sound, try VLC for Android or Youtube on your device to check.

Okay so looks like it is opening and connecting to the apk on the phone, but still not getting audio through VLC, could it possibly be some codec problem?

**Comment by rom1v** [comment]
Even with the youtube app or VLC for Android?

**Comment by jeongpup** [comment]
> Even with the youtube app or VLC for Android?

yeah even with those apps. I have a Samsung S9+ running on Andriod 10

**Comment by rom1v** [comment]
OK, maybe you'll get a hint in the output of `adb logcat`:

```bash
adb logcat -c  # clear logs
./sndcpy
adb logcat -d > logs.txt
```

It seems this new playback capture API is very restrictive and may also be even more restrictive on custom roms :confused:

**Comment by jeongpup** [comment]
This is what I got
[logs.txt](https://github.com/rom1v/sndcpy/files/5179233/logs.txt)
